modules/installing-gcp-post-cluster-creation.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * installing-gcp-user-defined-labels-and-tags.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-post-cluster-creation_{context}"]
+= Configuration of labels and tags for GCP after the cluster creation
```

Comment:
`...after cluster creation` is not required

Reply:
But this for the post cluster creation phase.

Reply:
You can mention that in the introductory statement. Heading focuses on the action unless there's a specific distinction to be made like "pre-cluster creation" and "post cluster creation" which I don't see here.

Reply:
That will make all the three new models with the same title. Since everything is for configuring labels and tags. 
And for the cluster creation phase and post cluster creation, can you suggest something?

Reply:
As I am not very sure about the technical context, I think post creation is during the deployment. Kindly check with SME as to what the user is trying to do here. That will give you an idea on how to reword this.

Reply:
`Configuration of labels and tags for GCP after the cluster creation` would mean updating tags or labels post cluster creation, which is not supported. Could this is updated to mean `querying or reading tags and labels configured during cluster installation`?

Reply:
Acknowledged.